html/pfappserver/lib/pfappserver/Form/Config/Provisioning/mobileconfig.pm

```diff
@@ -8,27 +8,90 @@ pfappserver::Form::Config::Provisioning - Web form for a switch
 
 =cut
 
+use pf::config;
 use HTML::FormHandler::Moose;
 extends 'pfappserver::Form::Config::Provisioning';
 with 'pfappserver::Base::Form::Role::Help';
 
+has_field 'company' =>
+  (
+   type => 'hidden',
+   value => $Config{'pki'}{'organisation'},
```

Comment:
If this is a config parameter that belongs in pf.conf, then this shouldn't be added to the config of the provisioner.

You should instead call the config hash in your code where you need this info